.pullapprove.yml

```diff
@@ -1,8 +1,10 @@
 approve_by_comment: true
-approve_regex: ^LGTM
+approve_regex: '^(lgtm|LGTM)'
```

Comment:
I think we should just stick to `LGTM` as `lgtm` usually used as approve by someone who is not a maintainer.